pkg/controller/dependences.go

```diff
@@ -455,6 +455,7 @@ func newFakeControl(kubeClientset kubernetes.Interface, informerFactory informer
 		CDCControl:         NewFakeTiCDCControl(),
 		TiDBControl:        NewFakeTiDBControl(kubeInformerFactory.Core().V1().Secrets().Lister()),
 		BackupControl:      NewFakeBackupControl(informerFactory.Pingcap().V1alpha1().Backups()),
+		ProxyControl:       NewFakeTiProxyControl(kubeInformerFactory.Core().V1().Secrets().Lister()),
```

Comment:
This `kubeInformerFactory.Core().V1().Secrets().Lister()` is useless for `ProxyControl`, and we can remove it.